templates/etc/logcheck/logcheck.conf.j2

```diff
@@ -55,13 +55,17 @@ SORTUNIQ=0
 # Controls the base directory for rules file location
 # This must be an absolute path
 
```

Comment:
Reason for changing it to `if logcheck_rulesdir is defined`??

For managing the readme please have a look: https://github.com/Oefenweb/ansible-updatedb/pull/2#issuecomment-129735419